people/hc2973.md

```diff
@@ -0,0 +1,8 @@
+layout: Huanyao Chen
+github_username: huc021
+last_name: chen
+first_name: huanyao
+permalink: /people/hc2973/
+
```

Comment:
Thanks @huc021. This metadata must be between '---'. You can use [mv2451.md](https://raw.githubusercontent.com/cumc-dbmi/g4000/master/people/mv2451.md) as a reference. Please fix it by adding commits to this same branch. (You won't have to create a new pull request.)